Defs/RecipeDefs/Recipes_Production.xml

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+	<RecipeDef ParentName="MakeStoneBlocksBase">
+		<defName>Make_FSXDrugLab</defName>
+		<label>Synthesize 10 FSX</label>
+		<description>Synthesize 10 FSX from chemfuel using simple chemistry equipment.</description>
+		<jobString>Synthesizing FSX</jobString>
+		<ingredients>
+		<li>
+		<filter>
+			<thingDefs>
+				<li>Chemfuel</li>
+			</thingDefs>
+		</filter>
+		<count>100</count>
+		</li>
+		</ingredients>
+		<fixedIngredientFilter>
+		<thingDefs>
+			<li>Chemfuel</li>		
+		</thingDefs>		
+		</fixedIngredientFilter>
+		<workAmount>1000</workAmount>
+		<workSkill>Crafting</workSkill>		
+		<workSpeedStat>DrugSynthesisSpeed</workSpeedStat>
+		<effectWorking>Smith</effectWorking>
+		<soundWorking>Recipe_Drug</soundWorking>	
+		<recipeUsers Inherit="false">
+			<li>DrugLab</li>
+		</recipeUsers>		
+		<products>
+			<FSX>7</FSX>
+		</products>
+	</RecipeDef>
+
+	<RecipeDef ParentName="MakeStoneBlocksBase">
+		<defName>Make_FSXRefinery</defName>
+		<label>Synthesize 10 FSX</label>
```

Comment:
Need to update recipe label and description from 10 to 8 (or however many units of FSX we ultimately decide on).